platform/src/common/components/Settings/API/EditClientForm.jsx

```diff
@@ -105,22 +119,22 @@ const EditClientForm = ({ open, closeModal, cIP = '', cName = '', clientID }) =>
 
       return () => clearTimeout(timer);
     }
-  }, [isError]);
+  }, [isError.isError]);
```

Comment:
Thanks @OchiengPaul442 :
- Is this variable "isError" always present?
- If not always available, we might want to handle this use case to minimise chances of runtime errors. There are couple of options which must be BACKWARD compatible with earlier version of node (JS).

Thanks!

Reply:
Hey @Baalmart ,
The state hook declaration you mentioned is used to manage the error state and display appropriate messages to the user when something goes wrong. This ensures that the error state is always present and updated accordingly whenever an error is detected.

You’re right about the compatibility. This state hook is specifically designed for use in a React frontend and won’t be compatible with a Node.js backend. However, it works perfectly fine for managing error states in a React component.
```
  const [isError, setIsError] = useState({
    isError: false,
    message: '',
    type: '',
  });
```